.circleci/unit_tests.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+export SHELL=/bin/bash
+
+set -e
+
+echo "Running Golang unit tests..."
+
+go get github.com/Sirupsen/logrus
```

Comment:
can vendor dependency be installed using a makefile or gopkg ~"install deps" command if these dependencies are part of plugin deps and not just needed when running unit-tests.